pkg/kubelet/metrics/metrics.go

```diff
@@ -50,6 +50,13 @@ var (
 			Help:      "Latency in microseconds to sync all pods.",
 		},
 	)
+	StartPodWorkerLatency = prometheus.NewSummary(
+		prometheus.SummaryOpts{
+			Subsystem: kubeletSubsystem,
+			Name:      "start_pod_worker_latency_microseconds",
+			Help:      "Latency in microseconds from seeing a pod to starting a worker.",
+		},
```

Comment:
Optional nit: bucketing per pod would make debugging a lot easier. If you really want an aggregate that's different.

```go
	StartPodWorkerLatency = prometheus.NewSummaryVec(
		prometheus.SummaryOpts{
			Subsystem: kubeletSubsystem,
			Name:      "start_pod_worker_latency_microseconds",
			Help:      "Latency in microseconds from seeing a pod to starting a worker.",
		},
        []string{"pod_name"}
	)
```